apps/backend/src/modules/projects/project.controller.ts

```diff
@@ -227,17 +227,11 @@ class ProjectController extends BaseController {
 	 */
 	private async findAll(
 		options: APIHandlerOptions<{
-			query: ProjectGetAllRequestDto;
+			query: null | ProjectGetAllRequestDto;
```

Comment:
Why not `undefined`?
```suggestion
			query: undefined | ProjectGetAllRequestDto;
```